wannierberri/__evaluate.py

```diff
@@ -154,7 +154,14 @@ def paralfunc(Kpoint,_system,_grid,npar_k,fftlib):
 
     if restart:
         try:
-            K_list=pickle.load(open(file_Klist,"rb"))
+            fr = open(file_Klist,"rb")
+            K_list = []
+            
+            while True:
+                try:
+                    K_list +=pickle.load(fr)
+                except:
```

Comment:
Maybe specify, what exception to expect. `EOFError` ?

Reply:
except here is working to break the while loop after reading Klist

Reply:
Yes, but there may be different exceptions. Say, if it is EOFError, then it is fine, it is just means 
but if there is another error (say, the object cannot be unpickled, because it was pickled because it is corrupt, I do not know why), than the code should raise it and exit.

Reply:
![Screenshot from 2021-09-14 20-56-06](https://user-images.githubusercontent.com/22172977/133317500-efad22fc-6378-46aa-90a0-d07c9c1e57ee.png)

Reply:
![Screenshot from 2021-09-14 20-57-43](https://user-images.githubusercontent.com/22172977/133317667-3eb22824-8512-4c6e-a00e-c5eb4fbb9a08.png)